src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -100,45 +106,51 @@ protected AbstractBuilding(Colony colony, BlockPos pos)
     public abstract int getMaxBuildingLevel();
 
     /**
-     * Add build to a mapping.
-     * <code>buildingClass</code> needs to extend {@link AbstractBuilding}
-     * <code>parentBlock</code> needs to extend {@link AbstractBlockHut}
+     * Add build to a mapping. <code>buildingClass</code> needs to extend
+     * {@link AbstractBuilding} <code>parentBlock</code> needs to extend
+     * {@link AbstractBlockHut}
      *
-     * @param name              name of building.
-     * @param buildingClass     subclass of AbstractBuilding, located in {@link com.minecolonies.colony.buildings}
-     * @param parentBlock       subclass of Block, located in {@link com.minecolonies.blocks}
+     * @param name
+     *            name of building.
+     * @param buildingClass
+     *            subclass of AbstractBuilding, located in
+     *            {@link com.minecolonies.colony.buildings}
+     * @param parentBlock
+     *            subclass of Block, located in {@link com.minecolonies.blocks}
      */
-    private static void addMapping(String name, Class<? extends AbstractBuilding> buildingClass, Class<? extends AbstractBlockHut> parentBlock)
+    private static void addMapping(String name, Class<? extends AbstractBuilding> buildingClass,
+            Class<? extends AbstractBlockHut> parentBlock)
     {
         if (nameToClassMap.containsKey(name) || classNameHashToClassMap.containsKey(buildingClass.getName().hashCode()))
         {
-            throw new IllegalArgumentException("Duplicate type '" + name + "' when adding AbstractBuilding class mapping");
-        }
-        else
+            throw new IllegalArgumentException("Duplicate type '" + name
+                    + "' when adding AbstractBuilding class mapping");
+        } else
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this "else" keyword to a new dedicated line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ARightCurlyBraceDifferentLineAsNextBlockCheck)